api/v1alpha1/jobset_types.go

```diff
@@ -74,6 +74,20 @@ type JobSetStatus struct {
 
 	// Restarts tracks the number of times the JobSet has restarted (i.e. recreated in case of RecreateAll policy).
 	Restarts int `json:"restarts,omitempty"`
+
+	// ReplicatedJobsStatus track the number of JobsReady for each replicatedJob.
+	// +optional
+	// +listType=map
+	// +listMapKey=name
+	ReplicatedJobsStatus []ReplicatedJobStatus `json:"ReplicatedJobsStatus,omitempty"`
+}
+
+// ReplicatedJobStatus defines the observed ReplicatedJobs Readiness.
+type ReplicatedJobStatus struct {
+	Name string `json:"name"`
+
+	// +kubebuilder:default=0
+	ReadyJobs int32 `json:"readyJobs"`
```

Comment:
we should also report the number of succeeded jobs, Kueue will need to add up ready + succeeded to determine if the jobset successfully started.

Reply:
Job Ready means: 
`podsCount := *(job.Spec.Parallelism)
if job.Spec.Completions != nil && *job.Spec.Completions < podsCount {
	podsCount = *job.Spec.Completions
}
if job.Status.Succeeded+ready >= podsCount {
       Job Is READY
}`
And if Job is ready we just increment the ReadyJobs

Reply:
I am not sure how this is related to the comment, I am saying we need to report the number of succeeded jobs in the status; for example, if a ReplicatedJob have 20 jobs, 10 active and 10 succeeded, out of the 10 active jobs only 5 are ready, then the ReplicatedJobsStatus will be:

Ready: 5
Succeeded: 10